src/scss/components/_icons.scss

```diff
@@ -0,0 +1,87 @@
+/*
+   Icons
+   ========================================================================== */
+
+
+[class^='c-icon '],
+[class^='c-icon-'],
+[class*=' c-icon '],
```

Comment:
Should be – I think this was only needed before when it was `icon` as it was more likely to be a part of a word.  `c-icon` is much less likely to accidentally be a part of another classname.